tests/fixtures.py

```diff
@@ -20,6 +20,35 @@
 }
 """
 
+sample_toml = """
+ecs_name = "ardere-test"
```

Comment:
Stupid Q: Do we want to include a comment here? I've seen a few folks that gleefully ignore documents but scan through test cases for examples.